src/ui/templates/insurance/eligibility/eligible-to-apply-online.njk

```diff
@@ -45,6 +45,16 @@
       }
     }) }}
 
+    {# temporary POST button whilst we develop account creation #}
+    {{ govukButton({
+      text: "Create an account",
+      classes: "govuk-button--secondary",
+      attributes: {
+        formaction: '/insurance/eligibility/eligible-to-apply-online/temp-create',
```

Comment:
can this be passed as a variable?

Reply:
I did consider this, I think it's overkill as it's a temporary thing and will be removed entirely when we have account creation and sign in working?